pandas/tests/io/json/test_readlines.py

```diff
@@ -179,3 +179,9 @@ def test_readjson_unicode(monkeypatch):
         result = read_json(path)
         expected = pd.DataFrame({"£©µÀÆÖÞßéöÿ": ["АБВГДабвгд가"]})
         tm.assert_frame_equal(result, expected)
+
+
+def test_readjson_bool_series():
```

Comment:
Can you move this to `test_pandas` and add `GH 31464` as a comment in the function body?

Reply:
Done